_rules/iframe-non-empty-title-attribute-cae760.md

```diff
@@ -0,0 +1,208 @@
+---
```

Comment:
I have no idea why Github's UI is showing it this way, but as you can see in the commit for it; this file was rename:
https://github.com/act-rules/act-rules.github.io/pull/1464/commits/df5af2c377205db209f2912b5dfd1d431c7f56ff

Reply:
The thing is that git has no real notion of "moving" (it deletes and recreate instead, including with `git mv`).
I think this is due to move + too many changes and then git algo consider that it is a different file, not the same, and there is a high risk that it will creep that way in the merge commit (i.e., we'll loose continuity after merge).

We had a similar issue last year (with a label rule, iirc), which we ended up solving by having a PR just to rename the file (so that git picks the correct move in all its logs), and then a PR for doing the actual changes 😞 See #904 and linked PRs/issues.

Reply:
OK I'll open a new PR just to rename the file. We'll have to merge that in before this one then.